ts/smelter-web-wasm/src/react/hooks/useSmelter.ts

```diff
@@ -18,10 +19,15 @@ export function useSmelter(): Smelter | undefined {
     return () => {
       cancel = true;
       void (async () => {
-        await promise.catch(() => {});
+        await promise.catch(console.error);
         await smelter.terminate();
       })();
     };
-  }, []);
+  }, [
+    options?.framerate,
```

Comment:
Looking at it now, those options need to be reconsidered, This API looks a bit wierd